gloo/docs/installation/quick_start.md

```diff
@@ -1,198 +1,325 @@
 ---
-title: Open Source Quick Start
-weight: -1
+title: Kubernetes - Installing Gloo
+weight: 2
 ---
 
-## Installing Gloo
+{{% notice note %}}
+To install Gloo Enterprise you need a License Key. If you don't have one, go to **https://solo.io/glooe-trial** and
+request a trial now. Once you request a trial, an e-mail will be sent to you with your unique License Key that you will
+need as part of installing Gloo.
+{{% /notice %}}
+
+{{% notice info %}}
+Each Key is valid for **31 days**. You can request a new key if your current key has expired.
+The License Key is required only during the installation process. Once you install, a `secret` will be created to hold
+your unique key.
+{{% /notice %}}
+
+If this is your first time running Gloo, you’ll need to download the command-line interface (CLI) called `glooctl` onto
+your local machine. You’ll use this CLI to interact with Gloo, including installing it onto your Kubernetes cluster.
+Directions on installing `glooctl` are [here](../install_glooctl).
+
+### Options to Install Gloo
 
 To install Gloo, you can use one of two options:
```

Comment:
So there are lots of options for installation in this guide. maybe we could put a header with bullet points about what the different sections are? Or maybe there's an automated way to generate an outline of the page?